cli/standalone/binary.rs

```diff
@@ -646,9 +646,15 @@ impl<'a> DenoCompileBinaryWriter<'a> {
     remote_modules_store.add_redirects(&graph.redirects);
 
     let env_vars_from_env_file = match cli_options.env_file_name() {
-      Some(env_filename) => {
-        log::info!("{} Environment variables from the file \"{}\" were embedded in the generated executable file", crate::colors::yellow("Warning"), env_filename);
-        get_file_env_vars(env_filename.to_string())?
+      Some(env_filenames) => {
+        let mut aggregated_env_vars = IndexMap::new();
+        for env_filename in env_filenames.iter().rev() {
+          log::info!("{} Environment variables from the file \"{}\" were embedded in the generated executable file", crate::colors::yellow("Warning"), env_filename);
+
+          let env_vars = get_file_env_vars(env_filename.to_string())?;
+          aggregated_env_vars.extend(env_vars);
+        }
+        aggregated_env_vars
```

Comment:
I think we might need a test for this case

Reply:
Thank you for the suggestion! I'll add another test in tests/integration/run_tests.rs to cover the case with two --env flags, ensuring proper variable overriding. Working on it now!

And I have noted to update the docs as well, should I do it after the merge or after the release.

Reply:
@bartlomieju , added more descriptive help text and also added the integration test which is working as expected, below is the result for your reference:
```bash
    running 3 tests
    test run::env_file ... ok
    test run::env_file_missing ... ok
    test run::env_file_multiple ... ok
    
    test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured; 1102 filtered out; finished in 0.78s
```